### PR TITLE
Added Greeting Github Actions

### PR DESCRIPTION
**Pull request by anushkrishnav**
## Description
Please include a summary of the change and which issue is fixed. Also include relevant motivation and context. List any dependencies that are required for this change.
Every Contributor to an OpenSource Project is special that is why I feel We should add a small GithubAction to give a warm welcome to the contributors.
Added a workflow file in .github folder with a greetings.yml file

## Contribution Guidelines
Please Read through the [Contribution Guidelines](https://github.com/aquasecurity/kube-hunter/blob/master/CONTRIBUTING.md).

## Fixed Issues

Please mention any issues fixed in the PR by referencing it properly in the commit message.
As per the convention, use appropriate keywords such as `fixes`, `closes`, `resolves` to automatically refer the issue.
Please consult [official github documentation](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) for details.

Fixes #(issue)
#381 

## "BEFORE" and "AFTER" output

To verify that the change works as desired, please include an output of terminal before and after the changes under headings "BEFORE" and "AFTER".

### BEFORE
Any Terminal Output Before Changes.

### AFTER
Any Terminal Output Before Changes.

## Contribution checklist
 - [x] I have read the Contributing Guidelines.
 - [x] The commits refer to an active issue in the repository.
 - [ ] I have added automated testing to cover this case.
 
## Notes
Please mention if you have not checked any of the above boxes.

### CODE REVIEW

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Hola! @${{ github.actor }} 🥳, You just created an Issue!🌟 Thanks for making the Project Better and empowering Learners'
+        pr-message: 'Submitted a PR already ?? That's amazing @${{ github.actor }} . Sit tight until one of our amazing maintainers review it. Make sure you read the contributing guide'
```
Comment (danielsagi) [inline .github/workflows/greetings.yml:13]: You can remove the "amazing" :)

Comment (danielsagi) [inline .github/workflows/greetings.yml:13]: You can remove the "amazing" :)

File: .github/workflows/greetings.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Hola! @${{ github.actor }} 🥳, You just created an Issue!🌟 Thanks for making the Project Better and empowering Learners'
```
Comment (danielsagi) [inline .github/workflows/greetings.yml:12]: Lets do: 
"Hola! @${{ github.actor }} 🥳, You've just created an Issue!🌟 Thanks for making the Project Better"

**Review by danielsagi — CHANGES_REQUESTED** [review]
Nice addition to the project, Just had 2 minor comments

File: .github/workflows/greetings.yml (hunk 1 above)
Comment (anushkrishnav) [reply .github/workflows/greetings.yml:13]: Suree

**Comment by anushkrishnav** [comment]
Can I Contribute it to other projects also?

**Comment by anushkrishnav** [comment]
please take a look at it @danielsagi

**Review by danielsagi — APPROVED** [review]

**Pull request merged**